tools/js/incrementVersion.js

```diff
@@ -0,0 +1,85 @@
+const fs = require('fs')
+const { join } = require('path')
+
+const root = join(__dirname, '..', '..')
+const data = join(root, 'data')
+const platforms = ['bedrock', 'pc']
```

Comment:
Only bedrock uses the YAML files right now so this list should only contain bedrock

Reply:
Re: documentation - fully agree. But where should this be documented? minecraft-data, node-minecraft-data, or bedrock-protocol. To me I feel bedrock-protocol makes the most sense as that is where someone would likely look, but could either be standalone or a link to bedrock-specific instructions in minecraft-data. WDYT?